web/src/app/util/TelTextField.tsx

```diff
@@ -45,7 +45,7 @@ export default function TelTextField(
   // check validation of the input phoneNumber through graphql
   const [{ data }] = useQuery({
     query: isValidNumber,
-    variables: { number: '+' + phoneNumber },
```

Comment:
Is the `+` sign w/country code no longer needed?

Reply:
This was a bug we found that currently exists in production. We're already appending a `+` on line 99 in the `handleChange` func, so it's been getting sent back to API as `++15555555`.